core-bundle/contao/themes/flexible/styles/login.css

```diff
@@ -83,8 +83,11 @@ body {
 }
 
 .login-providers .passkey button {
-	padding-left: 36px;
 	background-image: var(--icon-passkey);
+}
+
+.login-providers .tl_submit {
+	padding-left: 36px;
```

Comment:
This change implies that every login provider has an icon, which might not be the case. I would suggest to revert this and add the `padding-left: 36px;` in the `backend_login.html.twig` template. WDYT?

Reply:
Correct, fixed in 26361235f9.